lib/utils/merge/print-changeset-messages.ts

```diff
@@ -0,0 +1,29 @@
+import { ContentTypeProps } from 'contentful-management'
+import { createMessageLogStructure } from './create-merge-message'
+import { renderChangesetMessages } from './render-changeset-messages'
+import { ChangesetItem } from './types'
+
+export function printChangesetMessages(
+  targetModel: ContentTypeProps[],
+  items: ChangesetItem[]
+): string {
+  const messages = createMessageLogStructure(targetModel, items)
+  const addedString = renderChangesetMessages(messages.added)
+  const deletedString = renderChangesetMessages(messages.deleted)
+  const updatedString = renderChangesetMessages(messages.updated)
+
+  let response = ''
+
+  function maybeAdd(value?: string) {
```

Comment:
I had to think a bit at first, what you are doing here. Nice solution though 👍

Reply:
merci 🙃